test/Makefile

```diff
@@ -24,7 +24,7 @@ ARCHFLAGS ?=
 # Compiler and linker flags
 ASFLAGS ?= $(ARCHFLAGS)
 CFLAGS_EXTRA ?= -Wmissing-field-initializers -Wmissing-braces
-CFLAGS ?= $(ARCHFLAGS) -std=c90 -D_GNU_SOURCE -DWOLFHSM_SYMMETRIC_INTERNAL -Wall -Werror -Wno-cpp $(CFLAGS_EXTRA)
+CFLAGS ?= $(ARCHFLAGS) -std=c90 -D_GNU_SOURCE -Wall -Werror -Wno-cpp $(CFLAGS_EXTRA)
```

Comment:
Is _GNU_SOURCE actually used/needed?

Reply:
no idea, I didn't write this Makefile

Reply:
I think I might have added this early on but of course I have no recollection why... I"ll play with it and remove if unnecessary in a subsequent commit